### ISSUE TITLE
no info about models (resources)

### ISSUE THREAD

**Issue by kminami** [kminami/apib2swagger#45]
Now if I take the more isolated example (and without faulty JSONs) [11. Resource Model.md](https://github.com/apiaryio/api-blueprint/blob/master/examples/11.%20Resource%20Model.md), I see this in definitions:

```json
    "definitions": {
        "My Message": {},
        "My MessageModel": {
            "type": "object",
            "properties": {
                "class": {
                    "type": "array",
                    "items": {
                        "type": "string"
                    }
                },
                "properties": {
                    "type": "object",
                    "properties": {
                        "message": {
                            "type": "string"
                        }
                    }
                },
                "links": {
                    "type": "array",
                    "items": {
                        "type": "object",
                        "properties": {
                            "rel": {
                                "type": "string"
                            },
                            "href": {
                                "type": "string"
                            }
                        }
                    }
                }
            },
            "example": {
                "class": [
                    "message"
                ],
                "properties": {
                    "message": "Hello World!"
                },
                "links": [
                    {
                        "rel": "self",
                        "href": "/message"
                    }
                ]
            }
        }
    },
```

So there are both `"My Message": {},` and `"My MessageModel": {...`. I may be wrong but shouldn't it just be `"My Message":` but with all the properties?

Also, what concerns me much more, in this example the **Retrieve a Message [GET]** method refers to the **My Message** model in response:

```
### Retrieve a Message [GET]
At this point we will utilize our `Message` resource model and reference it in
`Response 200`.

+ Response 200

    [My Message][]
```

Shouldn't the schema be added into the response section too? Now there's only the example JSON there:

```json
"/message": {
            "get": {
                "responses": {
                    "200": {
                        "description": "This is the `application/vnd.siren+json` message resource representation.",
                        "headers": {},
                        "examples": {
                            "application/vnd.siren+json": {
                                "class": [
                                    "message"
                                ],
                                "properties": {
                                    "message": "Hello World!"
                                },
                                "links": [
                                    {
                                        "rel": "self",
                                        "href": "/message"
                                    }
                                ]
                            }
                        }
                    }
                },
...
```

_Originally posted by @Vanderhoof in https://github.com/LucyBot-Inc/api-spec-converter/issues/209#issuecomment-485444015_

**Comment by kminami** [comment]
@Vanderhoof, 

> So there are both "My Message": {}, and "My MessageModel": {.... I may be wrong but shouldn't it just be "My Message": but with all the properties?

Because there is two ways to define a resource level schema: Attribute section and Model section, so I decided to prefixing latter one with "Model" for ease implementing.
It might be unnecessary separation essentialy.

> Also, what concerns me much more, in this example the Retrieve a Message [GET] method refers to the My Message model in response:
> Shouldn't the schema be added into the response section too? Now there's only the example JSON there:

This was just a bug. I fixed this at v1.9.2.
Thank you!

**Comment by Vanderhoof** [comment]
@kminami thank you for the fix!
